### PR TITLE
Add is_nil support to Ets/Mnesia

### PR DESCRIPTION
**Pull request by TheFirstAvenger**
Initial pass at support for is_nil in Ets/Mnesia. Not positive this is the right approach, let me know if it is close.

### CODE REVIEW

File: lib/ash/filter/filter.ex [hunk 1]
```diff
@@ -732,6 +732,13 @@ defmodule Ash.Filter do
     end
   end
 
+  defp add_expression_part({:is_nil, field}, context, expression) do
```
Comment (zachdaniel) [inline lib/ash/filter/filter.ex:735]: Right now you'd do it with `filter(field: [is_nil: true])`. `filer(is_nil: :field)` looks really nice though, good addition.

**Comment by zachdaniel** [comment]
Looks good! Just need to fix the credo error and then its good to go 👍

File: lib/ash/filter/filter.ex (hunk 1 above)
Comment (TheFirstAvenger) [reply lib/ash/filter/filter.ex:735]: Good to know, added support for the `is_nil: true` version.

**Comment by TheFirstAvenger** [comment]
@zachdaniel good to go

**Comment by zachdaniel** [comment]
🚀 Thank you for your contribution! 🚀

**Pull request merged**